### PR TITLE
Various logging tweaks

### PR DESCRIPTION
**Pull request by kklin**

### CODE REVIEW

File: cluster/cluster.go [hunk 1]
```diff
@@ -429,10 +434,10 @@ func syncDB(cms []machine.Machine, dbms []db.Machine) syncDBResult {
 
 func (clst cluster) get() ([]machine.Machine, error) {
 	var cloudMachines []machine.Machine
-	for _, p := range clst.providers {
-		providerMachines, err := p.List()
+	for tag, inst := range clst.providers {
```
Comment (Baisang) [inline cluster/cluster.go]: Is this backwards? I thought `clst.providers` mapped an instance to a provider.

Comment (kklin) [inline cluster/cluster.go]: Yeah, I made it `inst, p`.. Although I think `instance` is a poor name for that struct.

**Comment by kklin** [comment]
Travis passed when I re-ran it.

**Comment by kklin** [comment]
ping @Baisang

File: cluster/digitalocean/digitalocean_test.go [hunk 2]
```diff
@@ -184,20 +184,6 @@ func TestList(t *testing.T) {
 	machines, err = doClust.List()
 	assert.Nil(t, machines)
 	assert.EqualError(t, err, "get public IP: no networks have been defined")
-
-	respBad := &godo.Response{
```
Comment (kklin) [inline cluster/digitalocean/digitalocean_test.go]: This was testing error handling for the old pagination logic

**Pull request merged**